domain/src/main/java/com/manjee/domain/FormulaType.kt

```diff
@@ -0,0 +1,38 @@
+package com.manjee.domain
+
+enum class FormulaType {
+    PLUS {
+        override fun calculate(num1: Int, num2: Int) = num1 + num2
+    },
+    MINUS {
+        override fun calculate(num1: Int, num2: Int) = num1 - num2
+    },
+    MULTIPLE {
+        override fun calculate(num1: Int, num2: Int) = num1 * num2
+    },
+    DIVIDE {
+        override fun calculate(num1: Int, num2: Int) : Int {
+            if (num2 == 0) {
+                throw IllegalArgumentException("값을 0으로 나눌 수 없음")
+            }
```

Comment:
```suggestion
        require(num2 == 0)  { "값을 0으로 나눌 수 없음" }
```
kotlin의 require를 활용해보세요 🙂